lib/renderer.js

```diff
@@ -919,9 +920,12 @@
 
         var settings = {
             scaleX: scalar,
-            scaleY: scalar
+            scaleY: scalar,
+            width: component.width,
```

Comment:
It seems kinda weird that the property on `settings` that is passed in to `getPixmapParams` is `width` and not `targetWidth` since that's the name that is used within generator ... but I realize this is both an existing API choice AND open to debate, I just felt like saying it.  `width` seems confusable with "current width".  🚶